### PR TITLE
First pull request

### PR DESCRIPTION
**Pull request by shubhcctech**

### CODE REVIEW

File: src/Circle.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+}
+ 
+void Circle::setCircumference() {
+    circumference = 2*3.14*radius;
+}
+ 
+double Circle::getArea() {
+    return area;
+}
+
```
Comment (ombothe08) [inline src/Circle.cpp:30]: remove this space

**Review by ombothe08 — CHANGES_REQUESTED** [review]
please do changes

File: src/Circle.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
+#include "../header/Circle.h"
+#include<cmath>
+using namespace std;
```
Comment (Coderanildangi) [inline src/Circle.cpp:3]: Do not use globally declare std namespace.

File: src/Elipse.cpp [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        cerr << "Dimentions are non-negative." << endl;
+    }
+}
+ 
+void Elipse::setArea() {
+    area = 3.14*minAxis*maxAxis;
+}
+ 
+void Elipse::setCircumference() {
+    circumference = 3.14 * (3 * (maxAxis + minAxis) - sqrt((3 * maxAxis + minAxis) * (maxAxis + 3 * minAxis)));
```
Comment (Coderanildangi) [inline src/Elipse.cpp]: Use comments wherever possible.

File: src/Triangle.cpp [hunk 4]
```diff
@@ -0,0 +1,39 @@
+#include "../header/Triangle.h"
+#include <cmath>
+using namespace std;
+ 
+Triangle::Triangle() : area(0), perimeter(0) {}
+ 
+void Triangle::setDimensions(int side1, int side2, int side3) {
+    if (side1 >= 0 && side2 >= 0&&side3>=0) {
+        this->side1 = side1;
```
Comment (Coderanildangi) [inline src/Triangle.cpp:9]: Use naming convention like mSide1 instead of side1...

**Review by Coderanildangi — CHANGES_REQUESTED** [review]

**Pull request merged**